src/main/java/cc/polyfrost/oneconfig/config/gson/gsoninterface/GsonContext.java

```diff
@@ -0,0 +1,122 @@
+package cc.polyfrost.oneconfig.config.gson.gsoninterface;
+
+/*
+ * This file is part of OneConfig.
+ * OneConfig - Next Generation Config Library for Minecraft: Java Edition
+ * Copyright (C) 2021, 2022 Polyfrost.
+ * Copyright (c) 2012, Brandon Mintern, EasyESI, Berkeley, CA
```

Comment:
nice small (c)

Reply:
also add the copyright for outside stuff (Brandon etc) to the `LICENSE NOTICE FOR ADAPTED CODE`